app/src/index.d.ts

```diff
@@ -130,6 +135,7 @@ type BuiltInRule = {
 	rule: TokenRule
 }
 
+// Case frames
```

Comment:
Instead of comments on these, would it be worth adding some of that context into the name? (since the comments don't follow the usage)

Reply:
Possibly? Not for this PR, but we can think more about that